src/app/home/partners/partners.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="container center-align">
+  <p class="">Our Partners</p>
+  <div class="row">
+    <div class="card-image col s12 m3" *ngFor="let partner of partners | async">
+      <a href="//{{partner.link}}">
```

Comment:
To bind attributes, use the following notation: [href]
Also, this should open in a new tab, so add: target="_blank" rel="noopener noreferrer"
This last attribute is to avoid memory leaks and security problems like phishing attacks.